Tests/LibCore/TestLibCoreSharedSingleProducerCircularQueue.cpp

```diff
@@ -34,9 +34,8 @@ TEST_CASE(simple_dequeue)
     for (int i = 0; i < test_count; ++i)
         (void)queue.enqueue(i);
     for (int i = 0; i < test_count; ++i) {
-        auto const element = queue.dequeue();
-        EXPECT(!element.is_error());
-        EXPECT_EQ(element.value(), i);
+        auto const element = MUST(queue.dequeue());
```

Comment:
This file didn't get any TRY_OR_FAIL treatment (just MUST)

Reply:
Huh! You're right, this one slipped through the cracks. Weird.

Reply:
Ah! Now I remember:
```
In file included from /home/user/workspace/serenity/./AK/DeprecatedString.h:9,
                 from /home/user/workspace/serenity/Userland/Libraries/LibCore/SharedCircularQueue.h:13,
                 from /home/user/workspace/serenity/Tests/LibCore/TestLibCoreSharedSingleProducerCircularQueue.cpp:7:
/home/user/workspace/serenity/./AK/Format.h: In instantiation of 'AK::ErrorOr<void> AK::__format_value(TypeErasedFormatParams&, FormatBuilder&, FormatParser&, const void*) [with T = Core::SharedSingleProducerCircularQueue<int>::QueueStatus]':
/home/user/workspace/serenity/./AK/Format.h:298:133:   required from 'AK::VariadicFormatParams<allow_debug_formatters, Parameters>::VariadicFormatParams(const Parameters& ...) [with AK::AllowDebugOnlyFormatters allow_debug_formatters = AK::Yes; Parameters = {char [65], int, Core::SharedSingleProducerCircularQueue<int>::QueueStatus}]'
/home/user/workspace/serenity/./AK/Format.h:569:72:   required from 'void AK::outln(FILE*, CheckedFormatString<Parameters ...>&&, const Parameters& ...) [with Parameters = {char [65], int, Core::SharedSingleProducerCircularQueue<int>::QueueStatus}; FILE = FILE; CheckedFormatString<Parameters ...> = Format::Detail::CheckedFormatString<char [65], int, Core::SharedSingleProducerCircularQueue<int>::QueueStatus>]'
/home/user/workspace/serenity/./AK/Format.h:596:98:   required from 'void AK::warnln(CheckedFormatString<Parameters ...>&&, const Parameters& ...) [with Parameters = {char [65], int, Core::SharedSingleProducerCircularQueue<int>::QueueStatus}; CheckedFormatString<Parameters ...> = Format::Detail::CheckedFormatString<char [65], int, Core::SharedSingleProducerCircularQueue<int>::QueueStatus>]'
/home/user/workspace/serenity/Tests/LibCore/TestLibCoreSharedSingleProducerCircularQueue.cpp:37:30:   required from here
/home/user/workspace/serenity/./AK/Format.h:288:15: error: 'struct AK::Formatter<Core::SharedSingleProducerCircularQueue<int>::QueueStatus, void>' has no member named 'parse'
  288 |     formatter.parse(params, parser);
      |     ~~~~~~~~~~^~~~~
/home/user/workspace/serenity/./AK/Format.h:289:22: error: 'struct AK::Formatter<Core::SharedSingleProducerCircularQueue<int>::QueueStatus, void>' has no member named 'format'; did you mean 'Formatter'?
  289 |     return formatter.format(builder, *static_cast<T const*>(value));
      |            ~~~~~~~~~~^~~~~~
      |            Formatter
```

Or more succinctly: `TRY_OR_FAIL` would try to `format` a `SharedSingleProducerCircularQueue::QueueStatus`, which is an `enum class` without a formatter, so this fails.

I'll add a TODO here, too.